<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1869</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 184, 184);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1828</width>
        <height>1518</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>1500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(226, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>381</width>
            <height>71</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;Britannic Bold&quot;;</string>
          </property>
          <property name="text">
           <string>Publish Your Blog!!!</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Email Id:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>1350</x>
            <y>100</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>1350</x>
            <y>140</y>
            <width>321</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;Ink Free&quot;;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>100</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>910</x>
            <y>100</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>150</y>
            <width>351</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>920</x>
            <y>150</y>
            <width>351</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>Sports</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>personal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Food</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Travel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fashion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Personal</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Content:</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>300</y>
            <width>1801</width>
            <height>1111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>1680</x>
            <y>1430</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Franklin Gothic Demi Cond&quot;;</string>
          </property>
          <property name="text">
           <string>PUBLISH</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>1732</x>
            <y>0</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Britannic Bold&quot;;</string>
          </property>
          <property name="text">
           <string>BACK</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Riya/test.qrc"/>
 </resources>
 <connections/>
</ui>
